research/src/pages/select/select.proposal.mdx

```diff
@@ -195,11 +195,18 @@ and is therefore considered to be invalid.
 - The currently selected `<option>`, or the first if one isn't selected, should be
   visible within the listbox. This may require moving the list to accomodate listbox positioning
   and available space.
-- The user can dismiss the listbox and remove the state of `open` from the `<select>` in any of the
+- The user can dismiss the listbox and remove the state of `open` from the `<select>` in either of the
   following manners:
-  - The user clicks outside of the listbox
-  - Presses the escape key
-  - Selects one, or more (if `multiple` is true), by clicking or hitting `enter` or `space` key
+  - Triggering any of the listbox's <a href="#light-dismiss">light dismiss</a> behaviors
+  - Selecting one, or more (if `multiple` is true), options by clicking or hitting the `enter` or `space` key
+
+### Light dismiss
+The listbox part has "light dismiss", behavior, defined as being dismissed (removing
+the `open` state of the `<select>`, in this case) by either of the following things:
+- The user presses the escape key.
+- Focus moves outside of the subtree of the listbox (because of either user interaction or script),
```

Comment:
If the user clicks on a non-selectable element outside of the subtree, would this switch the focus? I'm not sure it would in the browser, but it should also be a means of dismissal

Reply:
Focus would switch to the `<body>` element in this case.  I've updated it to call out both cases separately:
- User clicks in non-focusable thing outside the listbox: focus moves to `<body>` and light dismiss occurs.
- User clicks on non-focusable thing inside the listbox: focus moves to `<body>`, but light dismiss does not occur.